pkg/cmd/root/root.go

```diff
@@ -43,14 +44,16 @@ var (
 	cfgMdPrefix = cfg.GetCfgMdValPrefix()
 	cfgDnPrefix = cfg.GetCfgDnValPrefix()
 	defaultCfg  = map[string]interface{}{
-		gf.ConfigFileFlag:           cfg.GetDefaultCfgFileName(),
-		gf.MockDelayInSecFlag:       0,
-		gf.MockTriggerTimeFlag:      "",
-		gf.MockIPCountFlag:          2,
-		gf.SaveConfigToFileFlag:     false,
-		gf.Imdsv2Flag:               false,
-		gf.RebalanceDelayInSecFlag:  0,
-		gf.RebalanceTriggerTimeFlag: "",
+		gf.ConfigFileFlag:                cfg.GetDefaultCfgFileName(),
+		gf.MockDelayInSecFlag:            0,
+		gf.MockTriggerTimeFlag:           "",
+		gf.MockIPCountFlag:               3,
```

Comment:
Change the default value back to 2. No need to change it

Reply:
For all the comments that you added for this PR, are you planning to make those changes or already have those changes? I'm little bit confused with your comments